convex/expenses.ts

```
@@ -40,6 +25,56 @@ async function getCurrentUserId(ctx: any) {
+        break;
+      case "engineer":
+        totalBonuses.maintenanceReduction += totalMultiplier * moraleMultiplier;
+        break;
+      case "quality_control":
+        totalBonuses.qualityBoost += totalMultiplier * moraleMultiplier;
+        break;
+      case "cost_optimizer":
+        totalBonuses.costReduction += totalMultiplier * moraleMultiplier;
+        break;
+      case "manager":
+        const managerBonus = (totalMultiplier * 0.5) * moraleMultiplier;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)